### ISSUE TITLE
Status Error on "state" information

### ISSUE THREAD

**Issue by rriscado** [blindsidenetworks/scalelite#613]
When I run the command "**./bin/rake status**", I receive all servers state equal to disabled, but I have servers enabled.

If I run the command "**./bin/rake servers**", I receive the correct status (State = Enabled).

I'm using **scalelite v1.1.6** and **BBB 2.3.8**

![image](https://user-images.githubusercontent.com/14829973/126896762-ea4aab03-6032-44e2-9a62-9b73976bd529.png)

**Comment by ffdixon** [comment]
There is a pull request that may address this.

  https://github.com/blindsidenetworks/scalelite/pull/610

If you wanted to try this on your own setup for Scalelite and give feedback, that would be most welcome.

**Comment by rriscado** [comment]
Hi,

I apologize, but the problem was em my local environment.

I was using an old version (1.0.5) where I founded a problem.
Because that, I customized my status.rake and map this file on my systemd start comand (--mount).

After update to version 1.1.6, I didn't remember to remove this config.

I removed this confg now and I'm using the original status.rake from Scalelite 1.1.6.
It's working fine!

Sorry!

Thank you so much for help-me"
